### ISSUE TITLE
How can I access nested InfoWindow?

### ISSUE THREAD

**Issue by sebastian-negoro** [xkjyeah/vue-google-maps#474]
Hello. If I have a nested InfoWindow, how can I, for example, change the _opened_ property by clicking on marker in following code:

`<GmapMarker
                :key="index"
                v-for="(m, index) in markers"
                :position="m.position"
                :clickable="true"
                :draggable="true"
                @click="center=m.position"
              >
                <GmapInfoWindow :opened="false">
                </GmapInfoWindow>
              </GmapMarker>`

Thank you!

**Comment by tomearly** [comment]
@sebastian-negoro Did you find an answer to this?

**Comment by sebastian-negoro** [comment]
@tomearly Yes, I think it's a quite stupid solution, however it works for me.
My application deals with delivery orders, every of which is having a unique id. So when user opens InfoWindow, I write that id to array and then I simply check if there is an element in the array.

`<GmapInfoWindow 
    :opened="openedInfoWindowsCheck(m.id)"
    @closeclick="infoWindowClosed(m.id)"
></GmapInfoWindow>

infoWindowOpened(index) {
        this.$set(this.$data.openedInfoWindows, index, true);
      },
      infoWindowClosed(index) {
        this.$set(this.$data.openedInfoWindows, index, false);
      },
      openedInfoWindowsCheck(index) {
        if((this.openedInfoWindows[index] === undefined) || (this.openedInfoWindows[index] === false)) {
          return false;
        }
        return true;
      }`

**Comment by tomearly** [comment]
@sebastian-negoro Hi, yes I implemented a very similar solution.

The info window shows on click and closes.
If I click a GMapMarker, then click another GMapMarker the current open GMapInfoWindow changes as expected.

However if I close an GMapInfoWindow and try and reopen any other GMapInfoWindow it does not reappear as expected.

Did you experience anything like this at all?

I can make a fiddle when I get two minutes :)

**Comment by zashton** [comment]
@tomearly 
I wrote up an example for this package, check it out https://github.com/xkjyeah/vue-google-maps/blob/vue2/examples/infowindow.html .
The idea is removing the nested InfoWindow from the marker . Only use 1 InfoWindow for all your markers. At the onclick function of all your markers , rewrite the description of the InfoWindow and set the lat,lng of the InfoWindow to the same as your clicked marker to get the "always 1 InfoWindow open effect"

**Comment by tomearly** [comment]
That is great. I will align my example with that.

** Update ** I can confirm your example works well for me. Thank you.
